packages/cardano-services/docker-compose.yml

```diff
@@ -114,6 +114,15 @@ services:
       - LOGGER_MIN_SEVERITY=${LOGGER_MIN_SEVERITY:-info}
       - SERVICE_NAMES=${SERVICE_NAMES:-asset,chain-history,network-info,rewards,stake-pool,tx-submit,utxo}
       - USE_QUEUE=${USE_QUEUE:-false}
+    healthcheck:
```

Comment:
This is valid when not in control of the `Dockerfile`, but it's better to define it as a build definition where possible so it's always available when using the image standalone. https://docs.docker.com/engine/reference/builder/#healthcheck

Reply:
Change is applied here [964c1e2](https://github.com/input-output-hk/cardano-js-sdk/pull/390/commits/964c1e2c0b68edec9b24042ec4e0e927e4a49cd9)